Online/Resource/Lobby.cs

```diff
@@ -15,10 +15,10 @@ public class Lobby : OnlineResource
 
         public string[] mods = RainMeadowModManager.GetActiveMods();
         public static bool modsChecked;
-
+        public int maxPlayerCount;
```

Comment:
Looks like everything is already self contained in LobbyInfo. So I don't think this isn't needed anymore